### PR TITLE
added updated class diagram and inserted it in analysis.tex

### PR DESCRIPTION
**Pull request by Joergenson**

### CODE REVIEW

File: project_report/tex/analysis.tex [hunk 1]
```diff
@@ -15,3 +15,9 @@ \subsubsection{Sequence Diagrams}
 \subsubsection{Operation Contracts}
 
 \subsubsection{Updated UML Class Diagram}
+\begin{figure}[ht]
+\centering 
+\includegraphics[scale=0.3]{images/diagrams/updated_UML_Class_Diagram.drawio.png}
```
Comment (kkmp-dk) [inline project_report/tex/analysis.tex:20]: I think there is some spelling mistakes? and I think the correct way to spell them is after -> 
SubscribtionHandler -> SubscriptionHandler
machineDataColllector -> machineDataCollector

image
Can the return type be one of these? that is pointed to? 
or that does not matter?

Comment (Joergenson) [reply project_report/tex/analysis.tex:20]: machine state is a separate thing it is used to translate the current machine state

**Review by kkmp-dk — APPROVED** [review]

File: project_report/tex/analysis.tex [hunk 2]
```diff
@@ -15,3 +15,9 @@ \subsubsection{Sequence Diagrams}
 \subsubsection{Operation Contracts}
 
 \subsubsection{Updated UML Class Diagram}
```
Comment (Kechr) [inline project_report/tex/analysis.tex:17]: Write a description including:
  - What are we seeing here?
  - What can this be used to in our development?

**Review by Kechr — CHANGES_REQUESTED** [review]

File: project_report/tex/analysis.tex [hunk 3]
```diff
@@ -15,3 +15,20 @@ \subsubsection{Sequence Diagrams}
...
+\label{figure:updated_UML_class_diagram} 
+\end{figure}
+
+
+The updated UML class diagram illustrates the current system idea based on the 
+analysis of the system. Although this diagram only shows the 
+OPC-UA client, it still gives a good idea of how this part of the system is going
+to be, once implemented. \\
+
+By using the diagram in implementation, the process gets a good starting point
```
Comment (Kechr) [inline project_report/tex/analysis.tex]: "... in the implementation phase ..." 
"... , the group has a good starting point to expand on."

**Review by Kechr — CHANGES_REQUESTED** [review]

**Review by Kechr — APPROVED** [review]

**Pull request merged**